reana/reana_dev/run.py

```diff
@@ -144,10 +144,14 @@ def run_commands():
     help="Which examples to run? [default=DEMO]",
 )
 @click.option(
-    "--admin-email", required=True, help="Admin user email address",
```

Comment:
Are these black changes being reported locally or only in GitHub actions? If they aren't locally it might have to do with the fact that [we do not use the latest Black version](https://github.com/reanahub/pytest-reana/blob/master/setup.py#L37).